### ISSUE TITLE
U12 Penalty-Regelung

### ISSUE THREAD

**Issue by BerndKlostermeyer** [RaceHorology/RaceHorology#425]
U12 Penalty-Regelung gem. DSV-Reglement incl. Protokoll sollte ähnlich DSValpinX implementiert werden.

**Comment by mstenz** [comment]
Ja, das sind Interessante Sonderregelungen. In der Bayernliga gibts auch so eine Regel mit %. Das Problem ist dass solche Regeln immer mal wieder verändert werden oder sehr schwer nachzuvollziehen sind.
Vor allem dürfen diese Berechnungen auch nur nach einem kompletten Durchgang durchgeführt werden, das Programm lässt aber grundsützlich immer das hinzufügen von zeiten zu. 
Ich persönlich wäre dafür eine Funktion einzuführen die eine Grundsatzmanipulation von Zeiten erlaubt, also dass man z.b. einen Dialog hat mit allen zeiten und man kann nun berechnungen mit diesen Zeiten durchführen. Also bestimmte Zeiten markieren usw. und dann in einem  speziellen Menü manuell die benötigten Kalkulationen durchführen. 
Wichtig ist auch dass die Original-Zeit nie überschrieben wird damit solche Kalkulationen immer wiederholbar sind.

**Comment by BerndKlostermeyer** [comment]
Um es anfangs nicht zu kompliziert zu machen, wäre eine Funktion wie bei alpinX ausreichend. Man kann sich ja die DB einmal wegsichern, bevor man die "Verpenaltygung" durchführt. Zum Thema Veränderung der Regelungen, sollten sie die Rennverantwortlichen bestenfalls im Vorfeld über die machbare Umsetzung Gedanken machen und nicht die Entwickler einer neuen Regelung hinterherprogrammieren ....

Da ja noch eine Menge zu tun ist, würde ich für Penalty alpinX übergangsweise weiter nutzen. Die Startliste #426 ist hierbei entscheidend.

die U12-Penaltyregelung ist seit 5-6 Jahren unverändert. Wurde damals sogar auch für Schüler und aktive eingeführt und nur nach lautstarkem Protest zurückgenommen.

**Comment by mstenz** [comment]
`Zum Thema Veränderung der Regelungen, sollten sie die Rennverantwortlichen bestenfalls im Vorfeld über die machbare Umsetzung Gedanken machen und nicht die Entwickler einer neuen Regelung hinterherprogrammieren ....`

Da stimme ich definitiv zu.
